app/next-client-app/lib/concept-utils.ts

```diff
@@ -1,10 +1,11 @@
-// Method to add concepts to a scanreport result
+// Method to add concepts and permissions to a scanreport result
 export const addConceptsToResults = <
-  T extends { id: number; concepts?: Concept[] }
+  T extends { id: number; concepts?: Concept[]; permissions: string[] }
 >(
   scanReportsResult: T[],
   scanReportsConcepts: ScanReportConcept[],
-  concepts: Concept[]
+  concepts: Concept[],
+  permissions: { permissions: string[] }
```

Comment:
As before - should this be of type `Permission[]`?

https://github.com/Health-Informatics-UoN/CaRROT-Mapper/blob/8d9b83936c438415caabc1073306261c50e57fdb/app/next-client-app/types/permissions.ts